src/main/java/io/stargate/sgv2/jsonapi/service/operation/filters/table/codecs/JSONCodecRegistries.java

```diff
@@ -48,6 +48,10 @@ public abstract class JSONCodecRegistries {
               JSONCodecs.DURATION_FROM_STRING,
               JSONCodecs.TIME_FROM_STRING,
               JSONCodecs.TIMESTAMP_FROM_STRING,
+              // UUID codecs
+              JSONCodecs.UUID_FROM_STRING,
+              JSONCodecs.TIMEUUID_FROM_STRING,
```

Comment:
Although we won't allow creation of "timeuuid" type, we do allow read/write, for which codec needed

Reply:
Sorry, I may not know the context for this one.
Why do we not allow "timeuuid" type?

Reply:
I am not 100% sure; I think it may be considered confusing thing -- you can store time-based UUIDs in "uuid" column anyway.
We can add support easily but right now spreadsheet includes type as unsupported.

/cc @amorton